src/Infrastructure.VS/LanguageDetection/SonarLanguageRecognizer.cs

```diff
@@ -123,15 +123,23 @@ private static bool IsTypeScriptDocument(IEnumerable<IContentType> contentTypes)
         public AnalysisLanguage? GetAnalysisLanguageFromExtension(string fileName)
         {
             var extension = GetNormalizedExtention(fileName);
-
+            
             // ContentType for "js" is typescript we do manual check to be consistent with Detect method
             if (JavascriptSupportedExtensions.Contains(extension)) { return AnalysisLanguage.Javascript; }
```

Comment:
Final nitpicking: drop a line please, no "reason" anymore to have the return on the same line